streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -190,7 +181,7 @@ public class TaskManagerTest {
 
     @org.mockito.Mock
     private InternalTopologyBuilder topologyBuilder;
-    @Mock(type = MockType.DEFAULT)
+    @org.mockito.Mock
```

Comment:
Can we now import this and avoid the fully qualified name?

Reply:
@clolov If you resolve Ismaels comment, could you also please remove the `Mockito` prefix for other method calls like `verify()`?